src/test/groovy/de/dkfz/roddy/execution/jobs/cluster/lsf/LSFJobManagerSpec.groovy

```diff
@@ -16,182 +16,85 @@ import de.dkfz.roddy.execution.jobs.JobState
-        Method method = LSFJobManager.class.getDeclaredMethod("queryJobInfo", Map)
-        method.setAccessible(true)
-        Object parsedJson = new JsonSlurper().parseText(RAW_JSON_OUTPUT)
-        List records = (List) parsedJson.getAt("RECORDS")
-
-        when:
-        GenericJobInfo jobInfo = method.invoke(jm, records.get(0))
 
-        then:
-        jobInfo != null
-        jobInfo.tool == new File("ls -l")
-    }
-
-    void "queryJobInfo, bjobs JSON output without lists  "() {
-
-        given:
-        def parms = JobManagerOptions.create().build()
-        TestExecutionService testExecutionService = new TestExecutionService("test", "test")
-        LSFJobManager jm = new LSFJobManager(testExecutionService, parms)
-        Method method = LSFJobManager.class.getDeclaredMethod("queryJobInfo", Map)
-        method.setAccessible(true)
-        Object parsedJson = new JsonSlurper().parseText(RAW_JSON_OUTPUT_WITHOUT_LISTS)
+        Object parsedJson = new JsonSlurper().parseText(getResourceFile(resourceFile).text)
         List records = (List) parsedJson.getAt("RECORDS")
 
         when:
-        GenericJobInfo jobInfo = method.invoke(jm, records.get(0))
+        GenericJobInfo jobInfo = jm.convertJobDetailsMapToGenericJobInfoobject(records.get(0))
 
         then:
         jobInfo != null
-        jobInfo.tool == new File("ls -l")
+        jobInfo.jobID.toString() == "22005"
+        jobInfo.tool == expectedTool
+
+        where:
+        resourceFile                                     | expectedJobId | expectedTool
+        "queryExtendedJobStateByIdTest.json"             | "22005"       | new File("ls -l")
+        "queryExtendedJobStateByIdWithoutListsTest.json" | "22005"       | new File("ls -l")
+        "queryExtendedJobStateByIdEmptyTest.json"        | "22005"       | null
     }
 
-    void "queryJobInfo, bjobs JSON output empty  "() {
+    @CompileStatic
+    String zonedDateTimeToString(ZonedDateTime date) {
+        DateTimeFormatter.ofPattern('MMM ppd HH:mm').withLocale(Locale.ENGLISH).format(date)
+    }
 
-        given:
-        String emptyRawJsonOutput= '''
-        {
-            "COMMAND":"bjobs",
-            "JOBS":1,
-            "RECORDS":[
-                {
-                "JOBID":"22005",
-                }
-            ]
-        }
-        '''
-        def parms = JobManagerOptions.create().build()
-        TestExecutionService testExecutionService = new TestExecutionService("test", "test")
-        LSFJobManager jm = new LSFJobManager(testExecutionService, parms)
-        Method method = LSFJobManager.class.getDeclaredMethod("queryJobInfo", Map)
-        method.setAccessible(true)
-        Object parsedJson = new JsonSlurper().parseText(emptyRawJsonOutput)
-        List records = (List) parsedJson.getAt("RECORDS")
+    @CompileStatic
+    String localDateTimeToLSFString(LocalDateTime date) {
+        DateTimeFormatter.ofPattern('MMM ppd HH:mm').withLocale(Locale.ENGLISH).format(date) + " L"
```

Comment:
This correctly specifies that this is a LSF string, but note, that not all of these use "L" and some have no such suffix. Therefore, document the L (or whatever alternative is appropriate) and maybe rename to narrow down the type of LSF date-time string that is produced.